libbpfgo/selftest/ringbuffers/.gitignore

```diff
@@ -0,0 +1,2 @@
+self
+self.bpf.o
```

Comment:
depending on the response to my other comments about libbpf.a and vmlinux.h , maybe need to add them here as well